webapp/views/wpt-results.js

```diff
@@ -416,30 +406,9 @@ class WPTResults extends Pluralizer(WPTColors(WPTFlags(PathInfo(LoadingState(Tes
       // path => {type, file[, refPath]} simplification.
       manifest: Object,
       screenshots: Array,
-      selectedMetadata: {
-        type: Array,
-        value: [],
-        observer: 'clearSelectedCells',
-      },
-      selectedCells: {
-        type: Array,
-        value: [],
-      },
-      testPlural: {
-        type: String,
-        computed: 'computeTestPlural(selectedMetadata)',
-      },
-      isTriageMode: {
-        type: Boolean,
-        observer: 'isTriageModeUpdated',
-      }
     };
   }
 
-  isTriageModeUpdated(isTriageMode) {
-    this.reloadData();
```

Comment:
Looks like `reloadData` is no longer used. Maybe remove it?

Reply:
It's still used [elsewhere](https://github.com/web-platform-tests/wpt.fyi/blob/master/webapp/views/wpt-results.js#L1007)